dts/arm/nxp/nxp_rt5xx_common.dtsi

```diff
@@ -0,0 +1,487 @@
+		clocks = <&clkctl1 MCUX_FLEXCOMM2_CLK>;
+		label = "FLEXCOMM_2";
+		status = "disabled";
+	};
+
+	flexcomm3: flexcomm@109000 {
+		compatible = "nxp,lpc-flexcomm";
+		reg = <0x109000 0x1000>;
+		interrupts = <17 0>;
+		clocks = <&clkctl1 MCUX_FLEXCOMM3_CLK>;
+		label = "FLEXCOMM_3";
+		status = "disabled";
+	};
+
+	flexcomm4: flexcomm@122000 {
+		compatible = "nxp,lpc-flexcomm";
+		reg = <0x122000 0x1000>;
+		interrupts = <18 0>;
+		clocks = <&clkctl1 MCUX_FLEXCOMM4_CLK>;
+		label = "FLEXCOMM_4";
+		status = "disabled";
+	};
+
+	flexcomm5: flexcomm@123000 {
+		compatible = "nxp,lpc-flexcomm";
+		reg = <0x123000 0x1000>;
+		interrupts = <19 0>;
+		clocks = <&clkctl1 MCUX_FLEXCOMM5_CLK>;
+		label = "FLEXCOMM_5";
+		status = "disabled";
+	};
+
+	flexcomm6: flexcomm@124000 {
+		compatible = "nxp,lpc-flexcomm";
+		reg = <0x124000 0x1000>;
+		interrupts = <43 0>;
+		label = "FLEXCOMM_6";
+		clocks = <&clkctl1 MCUX_FLEXCOMM6_CLK>;
+		status = "disabled";
+	};
+
+	flexcomm7: flexcomm@125000 {
+		compatible = "nxp,lpc-flexcomm";
+		reg = <0x125000 0x1000>;
+		interrupts = <44 0>;
+		label = "FLEXCOMM_7";
+		clocks = <&clkctl1 MCUX_FLEXCOMM7_CLK>;
+		status = "disabled";
+	};
+
+	flexcomm8: flexcomm@209000 {
+		compatible = "nxp,lpc-flexcomm";
+		reg = <0x209000 0x1000>;
+		interrupts = <60 0>;
+		label = "FLEXCOMM_8";
+		clocks = <&clkctl1 MCUX_FLEXCOMM8_CLK>;
+		status = "disabled";
+	};
+
+	flexcomm9: flexcomm@20a000 {
+		compatible = "nxp,lpc-flexcomm";
+		reg = <0x20a000 0x1000>;
+		interrupts = <61 0>;
+		label = "FLEXCOMM_9";
+		clocks = <&clkctl1 MCUX_FLEXCOMM9_CLK>;
+		status = "disabled";
+	};
+
+	flexcomm10: flexcomm@20b000 {
+		compatible = "nxp,lpc-flexcomm";
+		reg = <0x20b000 0x1000>;
+		interrupts = <62 0>;
+		label = "FLEXCOMM_10";
+		clocks = <&clkctl1 MCUX_FLEXCOMM10_CLK>;
+		status = "disabled";
+	};
+
+	flexcomm11: flexcomm@20c000 {
+		compatible = "nxp,lpc-flexcomm";
+		reg = <0x20c000 0x1000>;
+		interrupts = <63 0>;
+		label = "FLEXCOMM_11";
+		clocks = <&clkctl1 MCUX_FLEXCOMM11_CLK>;
+		status = "disabled";
+	};
+
+	flexcomm12: flexcomm@20d000 {
+		compatible = "nxp,lpc-flexcomm";
+		reg = <0x20d000 0x1000>;
+		interrupts = <64 0>;
+		label = "FLEXCOMM_12";
+		clocks = <&clkctl1 MCUX_FLEXCOMM12_CLK>;
+		status = "disabled";
+	};
+
+	flexcomm13: flexcomm@20e000 {
+		compatible = "nxp,lpc-flexcomm";
+		reg = <0x20e000 0x1000>;
+		interrupts = <65 0>;
+		label = "FLEXCOMM_13";
+		clocks = <&clkctl1 MCUX_FLEXCOMM13_CLK>;
+		status = "disabled";
+	};
+
+	usbhs: usbhs@144000 {
+		compatible = "nxp,mcux-usbd";
+		reg = <0x144000 0x1000>;
+		interrupts = <50 1>;
+		num-bidir-endpoints = <6>;
+		status = "disabled";
+		label = "USBHS_0";
+	};
+
+	hs_lspi: spi@126000 {
+		compatible = "nxp,lpc-spi";
+		/* Enabling cs-gpios below will allow using GPIO CS,
+		 rather than Flexcomm SS */
+		/* cs-gpios = <&gpio1 14 GPIO_ACTIVE_LOW>,
+			<&gpio1 15 GPIO_ACTIVE_LOW>,
+			<&gpio1 16 GPIO_ACTIVE_LOW>,
+			<&gpio1 17 GPIO_ACTIVE_LOW>; */
```

Comment:
please document this on the DT bindings, not as a comment

Reply:
I am going to delete these lines of code. They are in place in case we want to switch to using GPIO instead of the Slave Select from the IP block.